quantities/measures/Measures.cs

```diff
@@ -59,6 +71,13 @@ namespace Quantities.Measures;
     public static Double ToSi(in Double value) => TLeft.ToSi(TRight.ToSi(in value));
     public static Double FromSi(in Double value) => TRight.FromSi(TLeft.FromSi(in value));
     public static String Representation { get; } = $"{TLeft.Representation}{narrowNoBreakSpace}{TRight.Representation}";
+    public static void Write(IWriter writer)
+    {
+        writer.Start("prod");
```

Comment:
_prod_ makes me think of the _production environment_. If it's not too costly, rename to _product_.

Reply:
Thanks for the incentive to think this over! I've gone with "quotient" and "product". Much better names :smile: